[15-09-22 04:02:52]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:04:47]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:05:23]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:12:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:15:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:05:30]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:12:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:15:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:05:38]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:12:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:15:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:06:03]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:06:07]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:07:27]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:07:35]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:07:37]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:07:59]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:08:05]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:08:07]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:08:23]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:08:29]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:113)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:09:07]: 	uncaughtException: (no error message)
Error
    at logErrorOnValidation (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:13:11)
    at Object.error (/home/abhinav/Desktop/test-deployment/dist/config/validate.js:16:115)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:475:36)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:25:23]: 	uncaughtException: error() must return an Error object
Error: error() must return an Error object
    at new module.exports (/home/abhinav/Desktop/test-deployment/node_modules/@hapi/hoek/lib/error.js:23:19)
    at module.exports (/home/abhinav/Desktop/test-deployment/node_modules/@hapi/hoek/lib/assert.js:21:11)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:481:17)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:25:38]: 	uncaughtException: error() must return an Error object
Error: error() must return an Error object
    at new module.exports (/home/abhinav/Desktop/test-deployment/node_modules/@hapi/hoek/lib/error.js:23:19)
    at module.exports (/home/abhinav/Desktop/test-deployment/node_modules/@hapi/hoek/lib/assert.js:21:11)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:481:17)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
[15-09-22 04:25:40]: 	uncaughtException: error() must return an Error object
Error: error() must return an Error object
    at new module.exports (/home/abhinav/Desktop/test-deployment/node_modules/@hapi/hoek/lib/error.js:23:19)
    at module.exports (/home/abhinav/Desktop/test-deployment/node_modules/@hapi/hoek/lib/assert.js:21:11)
    at Object.internals.finalize (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:481:17)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:268:30)
    at internals.Base.$_validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:328:26)
    at Object.internals.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:145:28)
    at Object.exports.entry (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/validator.js:27:30)
    at internals.Base.validate (/home/abhinav/Desktop/test-deployment/node_modules/joi/lib/base.js:548:26)
